hoomd/md/pair/pair.py

```diff
@@ -526,6 +527,95 @@ def __init__(self, nlist, default_r_cut=None, default_r_on=0.):
         self._add_typeparam(params)
 
 
+class Table(Pair):
+    """Tabulated pair potential.
+
+    Args:
+        nlist (`hoomd.md.nlist.NList`): Neighbor list
+        default_r_cut (float): Default cutoff radius :math:`[\\mathrm{length}]`.
+        default_r_on (float): Default turn-on radius :math:`[\\mathrm{length}]`.
+
+    `Table` specifies that a tabulated pair potential should be applied between
+    every non-excluded particle pair in the simulation in the range
+    :math:`[r_{\\mathrm{min}}, r_{\\mathrm{cut}})`
+
+    Note:
+        For potentials that diverge near r=0, to set *r_min* to a non-zero
+        value.
+
+    The force :math:`\\vec{F}` is:
+
+    .. math::
+        :nowrap:
+
+        \\begin{eqnarray*}
+        \\vec{F}(\\vec{r}) = & 0 & r < r_{\\mathrm{min}} \\\\
+                           = & F(r)\\hat{r}
+                             & r_{\\mathrm{min}} \\le r < r_{\\mathrm{max}} \\\\
+                           = & 0 & r \\ge r_{\\mathrm{max}} \\\\
+        \\end{eqnarray*}
+
+    and the potential :math:`V(r)` is:
+
+    .. math::
+        :nowrap:
+
+        \\begin{eqnarray*}
+        V(r) = & 0 & r < r_{\\mathrm{min}} \\\\
+             = & V(r)
+               & r_{\\mathrm{min}} \\le r < r_{\\mathrm{max}} \\\\
+             = & 0 & r \\ge r_{\\mathrm{max}} \\\\
+        \\end{eqnarray*}
+
+    where :math:`\\vec{r}` is the vector pointing from one particle to the other
+    in the pair.
+
+    Provide :math:`F(r)` and :math:`V(r)` on an evenly space set of grid points
+    points between :math:`r_{\\mathrm{min}}` and :math:`r_{\\mathrm{cut}}`.
+    `Table` linearly interpolates values when :math:`r` lies between grid points
+    and between the last grid point and :math:`r=r_{\\mathrm{cut}}`.  The force
+    must be specificed commensurate with the potential: :math:`F =
+    -\\frac{\\partial V}{\\partial r}`.
+
+    `Table` does not support energy shifting or smoothing modes.
+
+    Attributes:
+        params (`TypeParameter` [\
+          `tuple` [``particle_type``, ``particle_type``],\
+          `dict`]):
+          The potential parameters. The dictionary has the following keys:
+
+          * ``r_min`` (`float`, **required**) - the minimum distance to apply
+            the tabulated potential, corresponding to the first element of the
+            energy and force arrays :math:`[\\mathrm{length}]`.
+
+          * ``V`` ((*N*,) `numpy.ndarray` of ``numpy.float64``, **required**) -
+            the tabulated energy values :math:`[\\mathrm{energy}]`.
+
+          * ``F`` ((*N*,) `numpy.ndarray` of ``numpy.float64``, **required**) -
+            the tabulated force values :math:`[\\mathrm{force}]`. Must have the
+            same length as ``V``.
```

Comment:
```suggestion
          * ``V`` ((*N*,) `numpy.ndarray` of `float`, **required**) -
            the tabulated energy values :math:`[\\mathrm{energy}]`.

          * ``F`` ((*N*,) `numpy.ndarray` of `float`, **required**) -
            the tabulated force values :math:`[\\mathrm{force}]`. Must have the
            same length as ``V``.
```
This is equivalent and the way NumPy documents and recommends using for methods/functions such as `astype`.